src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ITupleExpression.cs

```diff
@@ -1337,5 +1337,347 @@ static void Main()
 
             VerifyOperationTreeAndDiagnosticsForTest<CastExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void TupleFlow_01()
+        {
+            string source = @"
+class C
+{
+    void M(bool b)
+    /*<bind>*/{
+        (int, int) t = (1, b ? 2 : 3);
```

Comment:
>(1, b ? 2 : 3) [](start = 23, length = 14)

It would be good to cover scenarios when control flow is not in the last element.